deepchecks/vision/suites/default_suites.py

```diff
@@ -25,9 +28,87 @@
 __all__ = ['train_test_validation', 'model_evaluation', 'full_suite', 'integrity_validation', 'data_integrity']
 
 
-def train_test_validation(**kwargs) -> Suite:
-    """Create a suite that is meant to validate correctness of train-test split, including integrity, \
-    distribution and leakage checks."""
+def train_test_validation(n_top_show: int = 5,
+                          label_properties: List[Dict[str, Any]] = None,
+                          image_properties: List[Dict[str, Any]] = None,
+                          sample_size: int = 10_000,
+                          random_state: int = None,
+                          **kwargs) -> Suite:
+    """Suite for validating correctness of train-test split, including integrity, \
+    distribution and leakage checks.
```

Comment:
Here and in return value, and same for model evaluation suite - can you take the same wording as in tabular? (*possibly update it a bit as for model evaluation we don't check for overfitting etc.)